src/ctap/mod.rs

```diff
@@ -14,7 +14,7 @@
 
 pub mod command;
 #[cfg(feature = "with_ctap1")]
-mod ctap1;
+pub mod ctap1;
```

Comment:
If you have the `derive` conditional on a feature, this `pub` can also be conditional.